authenticity-collector/src/channel-subscription/channel-subscription.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import * as fs from 'fs';
+import { ChannelClient, ChannelInfo } from 'iota-is-sdk';
+import { ClientConfig, ApiVersion } from 'iota-is-sdk';
+
+@Injectable()
+export class ChannelSubscriptionService {
+	private readonly logger: Logger = new Logger(ChannelSubscriptionService.name);
+
+	constructor(private configService: ConfigService) {}
+
+	async channelSubscription() {
+		const collectorIdPath = this.configService.get<string>('COLLECTOR_ID_PATH');
+		const collectorJson = fs.readFileSync(collectorIdPath, 'utf-8');
+		const collector = JSON.parse(collectorJson);
```

Comment:
This assume we store the identity with its secret keys in a plain json file. This is not very secure and we need to adjust in a follow up story.